### PR TITLE
Adding guidance for minor changes to the standard

### PR DESCRIPTION
**Pull request by kathrynmohror**
Proposal to add guidance to the governance document to describe the reduced process for minor ("typo", formatting) changes to the standard document. The idea is that these changes could be reviewed by the ASC chairs and release managers and simply included in the next release without requiring readings and voting.

### CODE REVIEW

**Comment by kathrynmohror** [comment]
Please use emoji reactions ON THIS COMMENT to indicate your position on this proposal.

* You do not need to vote on every proposal
* If you have no opinion, don't vote - that is also useful data
* If you've already commented on this issue, please still vote so
      we know your current thoughts
* Not all proposals solve exactly the same problem, so we may end
      up accepting proposals that appear to have some overlap
This is not a binding majority-rule vote, but it will be a very
significant input into the corresponding ASC decision.

Here are the meanings for the emojis:

* Hooray or Rocket: I support this so strongly that I
      want to be an advocate for it
* Heart: I think this is an ideal solution
* Thumbs up: I'd be happy with this solution
* Confused: I'd rather we not do this, but I can tolerate it
* Thumbs down: I'd be actively unhappy, and may even consider
      other technologies instead
If you want to explain in more detail, feel free to add another
comment, but please also vote on this comment.

File: pmix_governance.tex [hunk 1]
```diff
@@ -425,6 +425,58 @@ \subsubsection{ASC Secretaries}\label{asc-secretaries}}
...
+PMIx participants may propose minor changes to the document that 
+should not require the full process required for substantive changes. 
+Examples of these minor changes include fixes for typographical errors 
+(``typos''), missing or erroneous punctuation, and formatting changes.
+
+In the case minor changes are proposed, the proposer should 
+create an issue and pull request as described in Sections~\ref{initial-discussion}
+and \ref{proposed-modification} and then bring the
+changes to the attention of the ASC Co-Chairs and Release Managers.
+The Co-Chairs and Release Managers will evaluate the proposed
```
Comment (kathrynmohror) [inline pmix_governance.tex]: @SteVwonder says: Do we want to allow community members (or ASC members) to also chime in with concern over how substantial a change is? Or keep it limited to Co-Chairs and RMs?

Comment (kathrynmohror) [reply pmix_governance.tex]: @SteVwonder I added some text to clarify that everyone should look at it and comment. Does it address your concern?

**Comment by jjhursey** [comment]
Note to Reviewer: This PR builds off of PR #7. Until PR #7 is merged the diff will include the changes from that PR (they are separate commits to this branch). Once PR #7 is merged then we will rebase this PR.

File: pmix_governance.tex (hunk 1 above)
Comment (SteVwonder) [reply pmix_governance.tex]: LGTM!

**Comment by jjhursey** [comment]
PMIx ASC Q1'2020 Meeting:
 * The first vote passed: 10 yes / 0 no / 1 abstain

**Comment by jjhursey** [comment]
PMIx ASC 2Q 2020 Meeting:
 * Passed second vote: 11 Yes / 0 No / 0 Abstain / 3 not present for vote

**Pull request merged**